Explorer/Assets/DCL/AvatarRendering/DemoScripts/Systems/InstantiateRandomAvatarsSystem.cs

```diff
@@ -298,39 +312,55 @@ private void CreateAvatar(ICharacterControllerSettings characterControllerSettin
             else { transformComp.Transform.position = new Vector3(startXPosition + (avatarIndex * 2), 3, startZPosition); }
 
             transformComp.Transform.name = $"RANDOM_AVATAR_{avatarIndex}";
-
-            CharacterController characterController = transformComp.Transform.gameObject.AddComponent<CharacterController>();
-            characterController.radius = 0.4f;
-            characterController.height = 2;
-            characterController.center = Vector3.up;
-            characterController.slopeLimit = 50f;
-            characterController.gameObject.layer = PhysicsLayers.CHARACTER_LAYER;
-
+            
             HashSet<URN> wearablesURN = new HashSet<URN>();
             foreach (string wearable in wearables)
                 wearablesURN.Add(new URN(wearable));
 
-            var avatarShape = Profile.Create(
+            var profile = Profile.Create(
                 StringUtils.GenerateRandomString(5),
                 StringUtils.GenerateRandomString(5),
                 new Avatar(BodyShape.FromStringSafe(bodyShape), wearablesURN, WearablesConstants.DefaultColors.GetRandomEyesColor(), WearablesConstants.DefaultColors.GetRandomHairColor(), WearablesConstants.DefaultColors.GetRandomSkinColor()));
 
-            World.Create(avatarShape,
-                transformComp,
-                characterController,
-                new CharacterRigidTransform(),
-                new CharacterAnimationComponent(),
-                new CharacterEmoteComponent(),
-                new CharacterPlatformComponent(),
-                new StunComponent(),
-                new FeetIKComponent(),
-                new HandsIKComponent(),
-                new HeadIKComponent(),
-                new JumpInputComponent(),
-                new MovementInputComponent(),
-                characterControllerSettings,
-                new RandomAvatar()
-            );
+
+            if (networkAvatar)
+            {
+                World.Create(profile,
+                    transformComp,
+                    new CharacterAnimationComponent(),
+                    new CharacterEmoteComponent(),
+                    new RandomAvatar());
+                // new RemotePlayerMovementComponent(queuePool),
```

Comment:
What happened here?

Reply:
I forgot to cleanup! I wanted to create some avatars without the overhead systems (Physics and IKs) since this were causing degraded performance. In reality, remote avatars dont have these systems